schema.sql

```diff
@@ -16,4 +16,40 @@ CREATE TABLE IF NOT EXISTS animals (
 
 -- Add the species column to the animals table
 ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
\ No newline at end of file
+ADD COLUMN species VARCHAR(255);
+
+-- Create the owners table
+CREATE TABLE IF NOT EXISTS owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INT
+);
+
+-- Create the species table
+CREATE TABLE IF NOT EXISTS species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+-- Create a temporary table to store the updated animals data
+CREATE TABLE animals_temp AS
+SELECT
+    id,
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg,
+    CASE WHEN name LIKE '%mon' THEN 2 ELSE 1 END AS species_id
+FROM animals;
+
+-- Drop the old animals table
+DROP TABLE animals;
+
+-- Rename the temporary table to animals
+ALTER TABLE animals_temp RENAME TO animals;
+
+-- Add the owner_id column to the animals table
+ALTER TABLE animals
+ADD COLUMN owner_id INT,
+ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
```

Comment:
Great job @Ochiengsteven 🤝

- [ ] As stated in the project requirements, kindly add the `species_id` column.
![species](https://github.com/Ochiengsteven/vet_clinic/assets/63917725/190d4310-066a-4fc5-a39e-6f567319942f)

Reply:
Hello @1GORDON I already added the requested ID columns. Please recheck properly 
![id columns](https://github.com/Ochiengsteven/vet_clinic/assets/91716656/7142a514-fad4-443e-81c4-c52b888cb48d)

if you feel this aint enough, I'd appreciate a zoom call so that you can guide me through what to do